setup.py

```diff
@@ -66,7 +72,9 @@ def main(argv: List[str]) -> None:
         reqs = f.read()
         install_requires = reqs.strip().split("\n")
```

Comment:
I think we'll need to parse `install-requirements.txt` here to have this change propagate to the `setup.py` wheels build

Reply:
oh, forgot to update, good catch!